wd_demo.au3

```diff
@@ -1263,3 +1264,41 @@ Func SetupOpera($bHeadless)
 	Local $sCapabilities = _WD_CapabilitiesGet()
 	Return $sCapabilities
 EndFunc   ;==>SetupOpera
+
+Func SetupEdgeIEMode() ; this is for MS Edge IE Mode
+	; https://www.selenium.dev/documentation/ie_driver_server/#required-configuration
+	Local $sTimeStamp = @YEAR & '-' & @MON & '-' & @MDAY & '_' & @HOUR & @MIN & @SEC
+	_WD_Option('Driver', 'IEDriverServer.exe') ;
+	_WD_Option('Port', 5555)
+	_WD_Option('DriverParams', '-log-file="' & @ScriptDir & '\log\' & $sTimeStamp & '_WebDriver_EdgeIEMode.log" -log-level=INFO' & " -port=" & $_WD_PORT & " -host=127.0.0.1" )
+#cs
```

Comment:
should we remove the commented section ?

Reply:
I would keep the first line and remove the formatted version. That way it matches the other "setup" routines.

Reply:
what abou the `ie.edgepath` usage should we revert them back ?

Reply:
I would leave it there to show that it's an option that could be used if needed.

Reply:
Done
[SetupEdgeIEMode - cleanup](https://github.com/Danp2/au3WebDriver/pull/454/commits/defe8af3f738cdaa5398230325cc80d0e3224a92)